datumaro/plugins/cifar_format.py

```diff
@@ -0,0 +1,179 @@
+# Copyright (C) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import os.path as osp
+import pickle
+
+import numpy as np
+from datumaro.components.converter import Converter
+from datumaro.components.extractor import (AnnotationType, DatasetItem,
+    Importer, Label, LabelCategories, SourceExtractor)
+from datumaro.util.image import find_images
+
+
+class CifarPath:
+    BATCHES_META = 'batches.meta'
+    TRAIN_ANNOTATION_FILE = 'data_batch_'
+    IMAGES_DIR = 'images'
+    IMAGE_SIZE = 32
+
+CifarLabel = ['airplane', 'automobile', 'bird', 'cat',
+    'deer', 'dog', 'frog', 'horse', 'ship', 'truck']
+
+
+class CifarExtractor(SourceExtractor):
```

Comment:
As I understand, it works with python version of the format. I think we need to document it somewhere.